src/SablierV2MerkleStreamerFactory.sol

```diff
@@ -56,7 +56,18 @@ contract SablierV2MerkleStreamerFactory is ISablierV2MerkleStreamerFactory {
         returns (ISablierV2MerkleStreamerLL merkleStreamerLL)
     {
         // Hash the parameters to generate a salt.
-        bytes32 salt = keccak256(abi.encodePacked(initialAdmin, lockupLinear, asset, merkleRoot, expiration));
+        bytes32 salt = keccak256(
+            abi.encodePacked(
+                initialAdmin,
+                lockupLinear,
+                asset,
+                merkleRoot,
+                expiration,
+                keccak256(abi.encode(streamDurations)),
```

Comment:
There's no need to hash the `streamDurations`. ABI-encoding it is enough.

Reply:
Pretty sure this is correct but I will let you @andreivladbrg confirm before squashing & merging.